tests/generator/base.py

```diff
@@ -138,6 +139,9 @@ def __init__(self):
             'use-ntp': self.YAML_TRUE,
             'use-routes': self.YAML_TRUE,
         }
+        self.DEFAULT_IPV6_RA = {
+            'use-dns': self.YAML_TRUE,
+        }
```

Comment:
My bad, the `use-dns` was using boolean before, while switching to use tristate I probably missed it since the test still passes with the code in place, I will remove it in the next commit